content/docs/command-reference/exp/pull.md

```diff
@@ -54,8 +54,9 @@ all <abbr>cached</abbr> data associated with the experiments to DVC
   (first parents) starting from the `--rev` baseline. Give a negative value to
   include all first-parent commits (similar to `git log -n`).
 
-- `-f`, `--force` - rewrite the `experiment` commit if it already exists in the
-  local repo. Equivalent to `git push --force` (rewrites history)
+- `-f`, `--force` - rewrite the experiment if it already exists in the local
+  repo. Note that similarly to `git reset --hard`, this overwrites the local
+  history.
```

Comment:
Do you guys think this note is informative and useful? We can remove it if not.

Reply:
I think it's enough to mention that local experiments might get overwritten (if they have the same name/ref)

Reply:
I think it's too alarming, especially since we currently avoid explaining any of the git details. `-f` will only overwrite for the experiment refs, but this makes it sound like it could impact the user's branches.